### PR TITLE
Adjusted schema versions; adjusted test data to match.

### PR DESCRIPTION
**Pull request by tbitonti**
Adjusted schema versions to match the specification versions for schemas which were updated.  Adjusted test data to match.

### CODE REVIEW

**Comment by tbitonti** [comment]
For issue #32 "Missing schema version updates"

**Comment by ivargrimstad** [comment]
This is a bit big to review. Please split it into one PR per schema update. Also note that some of the schemas will not change in 10. E.g. there is no EJB 5.0, so that schema must stay as is.

File: xml/src/ejb-jar_5_0.xsds [hunk 1]
```diff
@@ -23,7 +23,7 @@
 	    xmlns:xsd="http://www.w3.org/2001/XMLSchema"
 	    elementFormDefault="qualified"
 	    attributeFormDefault="unqualified"
-	    version="4.0">
+	    version="5.0">
```
Comment (ivargrimstad) [inline xml/src/ejb-jar_5_0.xsds:26]: Nope, EJB is not updated for EE 10

**Review by ivargrimstad — CHANGES_REQUESTED** [review]
Please split into one PR per schema

**Pull request closed without merging**